cmd/vicadmin/server.go

```diff
@@ -166,36 +350,49 @@ func (s *server) stop() error {
 	return nil
 }
 
+// logout handler expires the user's session cookie by setting its creation time to the beginning of time
+func (s *server) logoutHandler(res http.ResponseWriter, req *http.Request) {
+	websession, _ := s.uss.cookies.Get(req, sessionCookieKey)
+	// ignore parsing/marshalling errors because we're parsing a hardcoded beginning-of-time string
+	beginningOfTime := time.Unix(0, 0).Format(time.RFC3339)
+	websession.Values[sessionCreationTimeKey] = beginningOfTime
+	if err := websession.Save(req, res); err != nil {
+		http.Error(res, "Failed to expire user session", http.StatusInternalServerError)
+		return
+	}
+	s.uss.Delete(websession.Values[sessionUsernameKey].(string))
+	http.Redirect(res, req, "/", http.StatusTemporaryRedirect)
```

Comment:
This redirect will in turn redirect to the login page - should we just send the user there directly?